gpkit/solution_array.py

```diff
@@ -189,15 +246,16 @@ def results_table(data, title, minval=0, printunits=True, fixedcols=True,
             s = k.str_without("models")
             if not sortbyvals:
                 decorated.append((model, b, (varfmt % s), i, k, v))
-            else:
-                decorated.append((model, np.mean(v), b, (varfmt % s), i, k, v))
+            else:  # for consistent sorting, add small offset to negative vals
+                val = np.mean(np.abs(v)) - (1e-9 if np.mean(v) < 0 else 0)
+                decorated.append((model, -val, b, (varfmt % s), i, k, v))
```

Comment:
-val so we don't have to reverse the below anymore, which keeps var order alphabetical...